R/app-server.R

```diff
@@ -301,39 +301,33 @@ app_server <- function(input, output, session) {
     ## Counts of individuals, specimens, and files
     output$nindividuals <- renderValueBox({
       valueBox(
-        length(
-          unique(
-            c(
-              indiv()$individualID,
-              biosp()$individualID,
-              manifest()$individualID
-            )
+        sum(!is.na(unique(
```

Comment:
Since we repeat this calculation in several places, I wonder if it would make sense to write a small function, something like this: 

```r
count_unique_ids <- function(...) {
  sum(!is.na(unique(c(...))))
}
```

Reply:
I had that same thought, but haven't tried it out just yet. I vaguely remember some issues popped up when I tried doing something like that in the dccmonitor app. I'll give it a whirl and see how it goes. I would much rather have a small, separate, testable function.